### PR TITLE
Update the link of Singularity

### PR DESCRIPTION
**Pull request by smy20011**
The old singularity website says that it will go away soon(http://singularity.lbl.gov/). I update this link to their new website.

### CODE REVIEW

**Comment by smy20011** [comment]
@AkihiroSuda Done :)

File: docs/index.html [hunk 1]
```diff
@@ -199,7 +199,7 @@ <h3 id="o1-images">(<code>O1</code>) Images</h3>
 result in an image that has <code>umoci</code>&rsquo;s hacks baked into it (if you run an
 image modified by an unprivileged user as a privileged user it should work
 normally). However, <code>umoci</code> supports the usage of the previously mentioned
-<a href="https://github.com/cyphar/rootlesscontaine.rs/tree/master/proto/rootlesscontainers.proto"><code>user.rootlesscontainers</code> <code>xattr</code> attribute</a>,
+<a href="https://github.com/rootless-containers/rootlesscontaine.rs/blob/f7b0f5486bb0e0be75771ce50c54471296f040eb/docs/proto/rootlesscontainers.proto"><code>user.rootlesscontainers</code> <code>xattr</code> attribute</a>,
```
Comment (AkihiroSuda) [inline docs/index.html]: https://github.com/rootless-containers/proto

Comment (AkihiroSuda) [reply docs/index.html]: and please update md

Comment (smy20011) [reply docs/index.html]: I didn't change the target of this link. This is a side-effect after I invoke make command in the root directory. It seems that in this [commit](https://github.com/rootless-containers/rootlesscontaine.rs/commit/d53c28456ceb7eb03a8a66508ed7828c59cb4a56). You updated the target of [rootlesscontainers-proto](https://github.com/rootless-containers/rootlesscontaine.rs/commit/d53c28456ceb7eb03a8a66508ed7828c59cb4a56#diff-cf6cc870629a01941a1e8057b57cc30fL127), but somehow hugo didn't update [this link](https://github.com/rootless-containers/rootlesscontaine.rs/commit/d53c28456ceb7eb03a8a66508ed7828c59cb4a56#diff-eae600270579316beb8b8d9b9a5732d4L202).

Comment (AkihiroSuda) [reply docs/index.html]: oh sorry

**Pull request merged**